strongbox-storage/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/SearchResults.java

```diff
@@ -43,15 +42,7 @@ public String toString()
         for (SearchResult artifact : results)
         {
             sb.append(artifact.getStorageId()).append(':').append(artifact.getRepositoryId()).append(' ');
-            sb.append(artifact.getGroupId()).append(':');
-            sb.append(artifact.getArtifactId()).append(':');
-            sb.append(artifact.getVersion()).append(':');
-            sb.append(artifact.getExtension());
-            if (!StringUtils.isEmpty(artifact.getClassifier()))
-            {
-                sb.append(':');
-                sb.append(artifact.getClassifier());
-            }
+            sb.append(artifact.getPath()).append(':');
```

Comment:
I supposed that `SearchResult` can be applied to any `Layout` type (Maven, Nuget, etc.)